.yarnrc

```diff
@@ -3,5 +3,5 @@
 
 
 registry "https://registry.npmjs.org/"
-lastUpdateCheck 1575256805000
-yarn-path ".yarn/releases/yarn-1.19.2.js"
+lastUpdateCheck 1584964403046
```

Comment:
Used 1.18 because I was running into this problem: https://github.com/yarnpkg/yarn/issues/7807.